### PR TITLE
Update customize-the-app-launcher.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the time taken for the customer tile to show for all users. It takes between 0-24 hours, depending on the users

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6506

File: microsoft-365/admin/manage/customize-the-app-launcher.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ You can add your own custom tiles to the app launcher that point to SharePoint s
   
 8. Select **Save changes** to create the custom tile. 
     
-    Your custom tile now appears in the app launcher on the **All** tab for you and your users. 
+    Your custom tile will appear in the next 24 hours in the app launcher on the **All** tab for you and your users.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/customize-the-app-launcher.md]:
```suggestion
    Your custom tile will appear within the next 24 hours in the app launcher on the **All** tab for you and your users. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@twernermsft Please merge this PR. Thanks

**Pull request merged**